cl/search/models.py

```diff
@@ -1837,11 +1837,10 @@ def filter(self, *args, **kwargs):
         citation_str = kwargs.pop("citation", None)
         if citation_str:
             try:
-                from cl.citations.find_citations import get_citations
+                from eyecite.find_citations import get_citations
```

Comment:
The only reason we put imports in spots like this is to avoid circular references. I imagine that by pulling eyecite into its own package, we've successfully broken the circle and it can go at the top of the file?

Reply:
Good point, I mistakenly thought that that was some sort of weird optimization to save on rarely used imports! 4bab3a0 moves it to the top of the file.